qiskit/synthesis/evolution/product_formula.py

```diff
@@ -6,7 +6,7 @@
 # obtain a copy of this license in the LICENSE.txt file in the root directory
 # of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
 #
-# Any modifications or derivative works of this code must retain this
+# typing.Any modifications or derivative works of this code must retain this
```

Comment:
a random "typing" error :)

Reply:
Interesting, this actually seems to cause the lint error:
> lint: commands[5]> python /home/vsts/work/1/s/tools/verify_headers.py qiskit test tools crates
qiskit/synthesis/evolution/product_formula.py failed header check because:
Header apache text string doesn't match:

Reply:
Whoops, I was using find+replace without sufficient caution 😉 Fixed!